groups/committee-steering/groups.yaml

```diff
@@ -24,4 +24,4 @@ groups:
     members:
       - fosterd@vmware.com
       - jberkus@redhat.com
```

Comment:
We talked about this and would like to leave Josh on the ML for the election officers in case there are any urgent Elekto issues that come up in Pacific time, since Ali and I are both in European time zones.